src/Network/Struct.php

```diff
@@ -111,9 +111,7 @@ public function addBigIntField(string $field): void {
 	 * @return bool
 	 */
 	public static function isValid(string $json): bool {
-		// TODO: replace with json_validate once we more to 8.3
-		$result = json_decode($json, true, static::JSON_DEPTH, static::JSON_FLAGS);
-		return !!$result;
+		return simdjson_is_valid($json);
```

Comment:
This also supports the depth parameter. Are you sure you don't want to pass static::JSON_DEPTH?

Reply:
Nice catch, fixed